### PR TITLE
Update Accordion Container Color - RSC-1689

### PR DESCRIPTION
**Pull request by avnvbhatta**
https://issues.sonatype.org/browse/RSC-1689

### CODE REVIEW

**Comment by EmilyBermudez88** [comment]
Are there more snapshots that should be updated here? - there's a bunch of snapshots with accordions in the background (ie. breadcrumb, search dropdown, etc)

**Comment by EmilyBermudez88** [comment]
> > Are there more snapshots that should be updated here? - there's a bunch of snapshots with accordions in the background (ie. breadcrumb, search dropdown, etc)
> 
> Added here: [9579f8c](https://github.com/sonatype/sonatype-react-shared-components/commit/9579f8c1ee8ffaac68affc078ab2411de964ebc1) Took almost 25 mins for the entire suite to run!

Ugh, mine's even worse! Sometimes an hour for just dark or light mode

File: gallery/visualtests/__image_snapshots__/nx-transfer-list-half-js-nx-transfer-list-half-when-filtered-gives-the-move-up-button-no-tooltip-of-its-own-1.png
Comment (EmilyBermudez88) [inline gallery/visualtests/__image_snapshots__/nx-transfer-list-half-js-nx-transfer-list-half-when-filtered-gives-the-move-up-button-no-tooltip-of-its-own-1.png:1]: Is this one just the tooltip change?

Comment (avnvbhatta) [reply gallery/visualtests/__image_snapshots__/nx-transfer-list-half-js-nx-transfer-list-half-when-filtered-gives-the-move-up-button-no-tooltip-of-its-own-1.png:1]: > Is this one just the tooltip change?

Looks like it. It's been reverted b91b0bc8a45788680e41d754f5730f78d9e74605

Comment (rpokorny) [reply gallery/visualtests/__image_snapshots__/nx-transfer-list-half-js-nx-transfer-list-half-when-filtered-gives-the-move-up-button-no-tooltip-of-its-own-1.png:1]: > Is this one just the tooltip change?

Idk for sure if it's just the tooltip, but the tooltip is different in an incorrect way, so it needs to be stabilized (probably with a wait) and re-run.

**Review by rpokorny — APPROVED** [review]

**Pull request merged**